instapipeline/QuantiusAnnotation.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module contains the QuantiusAnnotation class.
+"""
+
+from fishanno import BaseAnnotation
```

Comment:
Please update all imports to the new package name.

Reply:
Sorry about this! Did this and checked the demo notebooks again.